cfep-20.md

```diff
@@ -0,0 +1,97 @@
+
+<table>
+<tr><td> Title </td><td> Standard package split (dev / doc / dbg) </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Sylvain Corlay &lt;sylvain.corlay@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Oct 21, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a policy on how to split packages in multiple outputs for
+
+ - the main runtime requirements (shared objects, executables)
+ - the developer requirements (headers, static libraries, pkg-config and cmake artifacts)
```

Comment:
Could static libraries be separated from this? Given conda is primarily based around using shared libraries it would be nice to have the static libraries split into a separate package than the other things.

Reply:
Yeah that would align with [CFEP 18]( https://github.com/conda-forge/cfep/blob/84fc0ceae274d155639ef1561e54fd57b2b91335/cfep-18.md ) where static libraries need to be dropped or at least split out into separate packages.

Reply:
I don't think we need to split static libraries in another package than `-dev`, which is most typically not a runtime requirement.